### PR TITLE
adding jws profile and moving to informational

### PR DESCRIPTION
**Pull request by Razumain**
This update adds a JSON Web Signature profile to SVT.

Its a rather simple and straightforward new profile with no technical surprises.

The only issue for debate is the daring move to define a JOSE Header Parameter "svt" to be added to the IANA registry (https://www.iana.org/assignments/jose/jose.xhtml).

Do we believe that IETF would accept such registration based on this informational document?

### CODE REVIEW

File: draft-santesson-svt-jws.txt [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+
+   Each SVT is stored in its associated signature's "svt" header as
+   defined in Section 3.1.
+
+3.1.  "svt" Header Parameter
+
+   The "svt" (Signature Validation Token) Header Parameter is used to
+   contain an array of SVT tokens to support validation of the
+   associated signature.  Each SVT token in the array has the format of
+   a JWT as defined in [RFC7519] and is stored as a string object
```
Comment (martin-lindstrom) [inline draft-santesson-svt-jws.txt]: "and is stored as a string object without further wrapping or encoding"...

But a JWT is an JSON object, and you are talking about storing it as a string value. Wouldn't it be better to store it as an object, or if a string value is desired, use Base64-encoding?

File: draft-santesson-svt-jws.txt [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+          data is outside the scope of this specification.
+
+   *  hash - The hash over the JWS Payload data bytes (not its
+      base64url-encoded string representation).
+
+4.3.  Signer Certificate References
+
+   The SVT SHALL contain a CertReference claims object.  The type claim
+   of the CertReference claims object SHALL be either chain or
+   chain_hash`.
```
Comment (martin-lindstrom) [inline draft-santesson-svt-jws.txt]: chain_hash\`. Remove the \`.?

Comment (martin-lindstrom) [reply draft-santesson-svt-jws.txt]: Or is a \` missing?

**Review by martin-lindstrom — APPROVED** [review]
Just a question about format.

File: draft-santesson-svt-jws.txt (hunk 1 above)
Comment (martin-lindstrom) [reply draft-santesson-svt-jws.txt]: OK, I'll understand. The JWT **is** a string. Ignore the comment.

File: draft-santesson-svt-jws.txt (hunk 2 above)
Comment (Razumain) [reply draft-santesson-svt-jws.txt]: Fixed

File: draft-santesson-svt-jws.txt (hunk 1 above)
Comment (Razumain) [reply draft-santesson-svt-jws.txt]: I made some text changes to hopefully make this even more clear to the reader.

**Pull request merged**